kdl_parser/src/kdl_parser.cpp

```diff
@@ -129,7 +141,9 @@ KDL::RigidBodyInertia toKdl(urdf::InertialSharedPtr i)
 bool addChildrenToTree(urdf::LinkConstSharedPtr root, KDL::Tree & tree)
 {
   std::vector<urdf::LinkSharedPtr> children = root->child_links;
+#ifdef HAS_ROS
   ROS_DEBUG("Link %s had %zu children", root->name.c_str(), children.size());
```

Comment:
Given that we have overrides at the top for `ROS_ERROR` and `ROS_WARN`, it seems like we should have one for `ROS_DEBUG` as well.

Reply:
`ROS_DEBUG` is now forwarded to `stdout`.
I think I initially did not define it to not clutter the output, since it should not be `stderr` and since `stdout` cannot be inhibited without removing all other `stdout` from a process.